crossdock/server/server.go

```diff
@@ -77,6 +85,9 @@ func (s *Server) Start() error {
 			return s.doJoinTrace(ctx, req.(*tracetest.JoinTraceRequest))
 		})
 	})
+	// TODO test_driver host needs to be initialized before this
+	//mux.HandleFunc("/endtoend", s.eHandler.Trace)
```

Comment:
why do you need test driver hostname? this is the server, it should be passive/stateless, *called* by the driver